numerics/cbrt.cpp

```diff
@@ -0,0 +1,71 @@
+﻿
+#include "numerics/cbrt.hpp"
+
+#include <pmmintrin.h>
+
+#include <cstdint>
+#include <limits>
+
+namespace principia {
+namespace numerics {
+
+constexpr std::uint64_t C = 0x2A9F7893782DA1CE;
```

Comment:
No idea what this constant does and how it was computed.  Could we have references to some literature in this code?